Formula/ghc.rb

```diff
@@ -1,33 +1,19 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/8.10.7/ghc-8.10.7-src.tar.xz"
-  sha256 "e3eef6229ce9908dfe1ea41436befb0455fefb1932559e860ad4c606b0d03c9d"
-  # We bundle a static GMP so GHC inherits GMP's license
-  license all_of: [
-    "BSD-3-Clause",
-    any_of: ["LGPL-3.0-or-later", "GPL-2.0-or-later"],
-  ]
-  revision 1
+  url "https://downloads.haskell.org/~ghc/9.0.2/ghc-9.0.2-src.tar.xz"
```

Comment:
Is it time to do this already? `ghcup`'s recommended version is still 8.10, so I thought that's still the version that upstream recommends.

Reply:
Good question. I don't know the criteria of ghcup treat a GHC version as recommended.
IMO, GHC 9.0.2 is a bug fix release, supported by ghcup, hls and 9.0.2 [stackage nightly](https://www.stackage.org/nightly-2022-01-27) support more package than GHC 8.10.7 [LTS](https://www.stackage.org/lts-18.23), which means GHC 9.0.2 mature enough for more adoption.
Another benefit of GHC 9.0.2 is, GHC 9.x has better compiling performance then 8.10, it *may* reduce some  CI resource of Haskell projects